### PR TITLE
Added ImportMethod and EnterPrivateKey screens

### PR DESCRIPTION
**Pull request by e-molochko**
This change is [Reviewable](https://reviewable.io/reviews/ozalexo/chronomint-rn/19)

### CODE REVIEW

File: src/screens/ImportMethod/ImportMethod.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  Text,
+  TouchableOpacity,
+  View,
+} from 'react-native'
+import PropTypes from 'prop-types'
+import textConstants from '../../locales/en'
+import TextButton from '../../components/TextButton'
+import styles from './ImportMethodStyles'
+
+
```
Comment (ozalexo) [inline src/screens/ImportMethod/ImportMethod.js]: 3 empty lines. Please check your ESlint settings.

File: src/screens/ImportMethod/ImportMethod.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+const Method = ({image, label, onPress}) => {
+    return (
```
Comment (ozalexo) [inline src/screens/ImportMethod/ImportMethod.js]: Wrong indentation

File: src/screens/ImportMethod/ImportMethod.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+export default class ImportMethod extends PureComponent {
+  renderMethod = (importMethod) => (
+    <Method
+      key={importMethod.id}
+      {...importMethod}
+      onPress={this.props.onSelectImportMethod(importMethod)}
+    />
+  )
+
+  render() {
```
Comment (ozalexo) [inline src/screens/ImportMethod/ImportMethod.js]: It must be `render ()` (space between render and brackets).

File: src/screens/EnterPrivateKey/EnterPrivateKeyStyles.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import { StyleSheet } from 'react-native'
```
Comment (ozalexo) [inline src/screens/EnterPrivateKey/EnterPrivateKeyStyles.js:6]: Missing license text

File: storybook/stories/index.js [hunk 5]
```diff
@@ -361,3 +368,15 @@ storiesOf('Screens/Login', module)
   .add('Start Screen', () => (
     <ScreenStart />
   ))
+  
+
+storiesOf('Screens', module)
```
Comment (ozalexo) [inline storybook/stories/index.js]: This block should be merged with code at lines 360-370.
Chapter 'Screens/Login' should be used.

File: src/screens/Login/Start/StartStyles.js [hunk 6]
```diff
@@ -39,13 +39,11 @@ export default StyleSheet.create({
...
     textAlign: 'center',
   },
-  primaryButton: {
-    marginBottom: 20,
-  },
   orText: {
     alignSelf: 'center',
     color: colors.textOnPurple,
     fontSize: 16,
+    marginVertical: 20,
```
Comment (ozalexo) [inline src/screens/Login/Start/StartStyles.js]: Not sure, that it is a better way (instead of primaryButton's marginBottom.
If we will delete 'or' text, then layout will be broken in this case. In previous variant it may be safely removed.

File: src/screens/ImportMethod/ImportMethodContainer.js [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+    navigate('SetAccountPassword')
+  }
+
+  handleSelectImportMethod = ({ screen, title }) => () => {
+    const { navigate } = this.props.navigation
+    navigate(screen)
+  }
+
+  render() {
+    return (
```
Comment (ozalexo) [inline src/screens/ImportMethod/ImportMethodContainer.js:23]: Do we really need render method in container? 
Maybe it is enough just to wrap ImportMethod as it was done in `src/screens/Login/Start/StartContainer.js`?

Also pressing on buttons in StoryBook should not fail app (red screen with error). All methods must have default values like `() => {}`

File: src/screens/ImportMethod/ImportMethod.js [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+  View,
+} from 'react-native'
+import PropTypes from 'prop-types'
+import textConstants from '../../locales/en'
+import TextButton from '../../components/TextButton'
+import styles from './ImportMethodStyles'
+
+
+
+const Method = ({image, label, onPress}) => {
```
Comment (ozalexo) [inline src/screens/ImportMethod/ImportMethod.js]: `image` is missing in PropTypes.

File: src/screens/EnterPrivateKey/EnterPrivateKey.js [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+
+    return (
+      <View style={styles.screenView}>
+        <Input
+          label='Private key'
+          onChangeText={onChangePrivateKey}
+          error={error}
+        />
+        <PrimaryButton
+          style={{ marginTop: 40 }}
```
Comment (ozalexo) [inline src/screens/EnterPrivateKey/EnterPrivateKey.js]: Please do not use inline styles

**Review by ozalexo — CHANGES_REQUESTED** [review]

File: src/screens/Login/Start/StartStyles.js (hunk 6 above)
Comment (e-molochko) [reply src/screens/Login/Start/StartStyles.js]: At this case we just need to set one property to one element,instead of setting one margin property for both buttons around "or" text

File: package.json [hunk 10]
```diff
@@ -8,9 +8,9 @@
     "start": "node node_modules/react-native/local-cli/cli.js start",
     "test": "jest",
     "postinstall": "bash -c './scripts/postinstall.sh' && yarn patch-package",
-    "storybook": "NODE_ENV=development BABEL_ENV=development storybook start -p 7007 -c .storybook -h 0.0.0.0 | react-native start --reset-cache --projectRoot storybook",
+    "storybook": "storybook start -p 7007 -c .storybook -h 0.0.0.0 | react-native start --reset-cache --projectRoot storybook",
```
Comment (ozalexo) [inline package.json]: What is the reason of this change?

File: src/screens/EnterPrivateKey/EnterPrivateKey.js [hunk 11]
```diff
@@ -0,0 +1,41 @@
...
+import PropTypes from 'prop-types'
+import Input from '../../components/Input'
+import PrimaryButton from '../../components/PrimaryButton'
+import styles from './EnterPrivateKeyStyles'
+
+export default class EnterPrivateKey extends PureComponent {
+
+  render() {
+    const {
+      onChangePrivateKey,
```
Comment (ozalexo) [inline src/screens/EnterPrivateKey/EnterPrivateKey.js]: Here is a  good place to define default values.
```
const {
  onChangePrivateKey = () => {},
. . . 
```

File: src/screens/EnterPrivateKey/EnterPrivateKeyStyles.js (hunk 4 above)
Comment (ozalexo) [reply src/screens/EnterPrivateKey/EnterPrivateKeyStyles.js:6]: Still missing license text

File: src/screens/EnterPrivateKey/index.js [hunk 12]
```diff
@@ -0,0 +1 @@
+export default from './EnterPrivateKeyContainer'
```
Comment (ozalexo) [inline src/screens/EnterPrivateKey/index.js:6]: Missing license text.

Check all changed files please.

File: src/screens/Login/Start/StartStyles.js (hunk 6 above)
Comment (ozalexo) [reply src/screens/Login/Start/StartStyles.js]: Yes, but as I described above, we will need to modify multiple elements in case of changes of 'or' text block.
From my point of view, each block should be easily removable with no side-effects to layout of other neighbour elements.

**Review by ozalexo — APPROVED** [review]
